src/bzflag/GuidedMissleStrategy.cxx

```diff
@@ -13,6 +13,10 @@
 /* interface header */
 #include "GuidedMissleStrategy.h"
 
+/* System headers */
+#define GLM_ENABLE_EXPERIMENTAL
```

Comment:
I want to put there because I want to remember where it is needed, if glm declare as a stable feature, or rewrite if is deprecated

Reply:
My point is that all that is needed to subvert your intention is for some file included before this point to include the glm header without the #define and this won’t do what you want. That’s why I say it’s error prone

Reply:
My point is that all that is needed to subvert your intention is for some file included before this point to include the glm header without the #define and this won’t do what you want. That’s why I say it’s error prone

Reply:
Understand.
However this is just enabling some API, not changing the behaviour.  If some will include the glm header file before this will not compile anymore.
And I don't like to enable experimental all over. Better that people think abouth before using it.
Dunno